integration-libs/cdc/root/consent-management/converters/cdc-user-preference.serializer.ts

```diff
@@ -7,6 +7,10 @@
 import { Injectable } from '@angular/core';
 import { ConsentTemplate, Converter } from '@spartacus/core';
 
+/**
+ * @deprecated since 2211-ng19.0, use class CdcPreferenceSerializer instead
```

Comment:
FYI It's still debatable whether the Feb release will be named `2211-ng19.0`. We're checking now if it's possible, due to being incompatible with the SemVer schema https://semver.org/ - which is assumed in the npm ecosystem

Reply:
done, i have changed this to 2211.21